src/Penrose/Optimizer.hs

```diff
@@ -79,8 +79,8 @@ useAutodiff = True
 constT :: Floating a => a
 constT = 0.001
 
--- debugOpt = True
-debugOpt = False
+debugOpt = True
```

Comment:
Revert to `False` for deployment: Console IO affects performance quite negatively